frontend/survey-project/src/app/components/shared/loader/loader.component.css.map

```diff
@@ -0,0 +1 @@
+{"version":3,"sources":["loader.component.scss","loader.component.css"],"names":[],"mappings":"AAAA;EACI,WAAA;EACA,YAAA;EACA,aAAA;EACA,qCAAA;ACCJ;;ADEC;;EAEG,WAAA;EACA,cAAA;EACA,oBAAA;EACA,kBAAA;EACA,+DAAA;EACA,sBAAA;EACA,4CAAA;ACCJ;;ADEC;EACG,+DAAA;EACA,4BAAA;ACCJ;;ADEC;EACG;IACG,wBAAA;ECCL;AACF","file":"loader.component.css"}
```

Comment:
Not sure what is this used for, but I think it's not needed, we're using scss not css and the Angular builder will build our scss to css.